Formula/tmux.rb

```diff
@@ -32,11 +34,16 @@ class Tmux < Formula
   def install
     system "sh", "autogen.sh" if build.head?
 
+    configure_args = [
+      "--disable-Dependency-tracking",
+      "--prefix=#{prefix}",
+      "--sysconfdir=#{etc}",
+    ]
```

Comment:
> For things like deciding which way to define Arrays of strings, you can absolutely alter Rubocop to enforce the style of the repo.

This does not conflict with "I doubt you can ask rubocop to flag configure_args in favor of args".

As for the way to define arrays of strings, I was talking specifically about the convention for defining the `args` array to `configure` (although I do believe `%W[]` or `%w[]` are preferred). Conventions aren't necessarily rules, and don't necessarily require enforcement, but it's nice to follow them for consistency.

> Knowing that the contribution guidelines require you to run rubocop to enforce style, and that this PR passed that check, do you feel this is absolutely a blocker on the PR?

I never said it's a blocker, and I only left a comment, not a "request changes". By the way I'm not even a maintainer at this point, so you can always ignore my opinions. But what's so hard about taking a (friendly, or at least neutral) advice? Why are you offended, if I'm reading you correctly?